src/core_modules/capture-core/components/WidgetEventEdit/EditEventDataEntry/EditEventDataEntry.component.js

```diff
@@ -160,17 +168,14 @@ const buildScheduleDateSettingsFn = () => {
             ),
         );
     const scheduleDateSettings = {
-        getComponent: () => scheduleDateComponent,
+        getComponent: (props: Object) => scheduleDateComponent(props),
         getComponentProps: (props: Object) => createComponentProps(props, {
             width: '100%',
             calendarWidth: 350,
             label: props.formFoundation.getLabel('scheduledAt'),
             disabled: true,
         }),
-        getIsHidden: (props: Object) => {
-            const isScheduleableStatus = [statusTypes.SCHEDULE, statusTypes.OVERDUE].includes(props.eventStatus);
-            return props.hideDueDate || !isScheduleableStatus;
-        },
+        getIsHidden: (props: Object) => props.hideDueDate,
```

Comment:
Hey, 

Should the Scheduled date also be shown in Event programs? If not one way to fix it is to check that the event belongs to a Tracker program by looking at the `props.id`. Something like this: `getIsHidden: (props: Object) => props.id === dataEntryIds.ENROLLMENT_EVENT && props.hideDueDate`

<img width="440" alt="Screenshot 2022-10-18 at 09 21 53" src="https://user-images.githubusercontent.com/6817073/196363362-4466ea22-f3d4-47b6-985b-d8f834b4db90.png">

Thanks!

Reply:
hey good catch @simonadomnisoru, I updated the branch to reflex the changes now. thanks for looking into it

Reply:
Hey,
I re-tested and I can see still the scheduled date in the Event programs. The logic I previously suggested was wrong 😓
I believe something like this should work better:
`getIsHidden: (props: Object) => props.id !== dataEntryIds.ENROLLMENT_EVENT || props.hideDueDate`. 

WDYT? Thanks! 🙏

Reply:
thanks @simonadomnisoru! I verified it should be hidden now on the view event page
<img width="1053" alt="Screenshot 2022-10-19 at 09 09 06" src="https://user-images.githubusercontent.com/89806888/196634003-7dfc6552-40be-4959-9ee3-9640675e1b30.png">